### DISCUSSION TITLE
How to restore gogs backup.

### DISCUSSION THREAD

**Discussion by waleedsaeed17** [gogs/gogs#7435] [Q&A]
facing issues in restoring GOGs backup,
I installed gogs in a new ubuntu VM, configured database schema named gogs, but dont know how to restore previous gogs data to this newly installed gogs?
kindly help!!

**Comment by annpocoyo** [comment]
Do you have database files and files from directory custom?

**Reply by waleedsaeed17** [reply]
Can you tell me the correct way to properly backup the GOGS either it be a database(MySQL) and configuration files from custom directories and then how to restore them??

**Comment by annpocoyo** [comment]
To restore your gogs installation, all you have to do is:
1. Restore the database.
2. Reinstall gogs how you usually do.
3. Restore the configuration files (Change them if necessary).
4. Restore your git-repositories directory and put it where you want (You may need to change your config).
5. Run gogs as normal.
